R/prettyRainfallPlot.R

```diff
@@ -125,19 +129,14 @@ prettyRainfallPlot = function(this_sample_id,
     this_maf = suppressMessages(read_tsv(maf_path))
     if(!missing(this_sample_id)){
       this_maf = this_maf %>% dplyr::filter(Tumor_Sample_Barcode %in% this_sample_id)
-    }else{
-      this_sample_id = "all samples"
     }
   } else if(!missing(this_sample_id)) {
     message ("MAF df or path to custom MAF file was not provided, getting SSM using GAMBLR ...")
-    these_ssm = get_ssm_by_sample(this_sample_id,
+    these_ssm = get_ssm_by_sample(this_sample_id = this_sample_id,
                                   projection = projection, 
-                                  this_seq_type = this_seq_type)
+                                  this_seq_type = this_seq_type, )
```

Comment:
Did you mean to add this comma? Doesn't look like there is an additional argument following

Reply:
Good find, thanks. I have corrected this and pushed to this PR